init-server.h

```diff
@@ -0,0 +1,64 @@
+#ifndef __SOCKETS__
+#define __SOCKETS__
+
+#define MAX_SOCKET_BUFF_SIZE 1024u
+
+WiFiServer wifiServer(80);
+char socketBuff[MAX_SOCKET_BUFF_SIZE] = {0};
+bool isAPMode = false;
+uint8_t requestHeader;
```

Comment:
Default value of this variable is set to 0. What about explicit assignment to 0, instead of implicit.

Reply:
what does that mean

Reply:
This means to write requestHeader = 0.